examples/cli_example_tiny_ephys_training.py

```diff
@@ -61,41 +53,28 @@
 generator_param["pre_post_omission"] = 1
 
 # Those are parameters used for the network topology
-network_param["type"] = "network"
 network_param[
     "name"
 ] = "unet_single_ephys_1024"  # Name of network topology in the collection
 
 # Those are parameters used for the training process
-training_param["type"] = "trainer"
 training_param["name"] = "core_trainer"
 training_param["loss"] = "mean_absolute_error"
 
 training_param["model_string"] = (
     network_param["name"]
     + "_"
     + training_param["loss"]
-    + "_"
-    + run_uid
 )
-training_param["output_dir"] = "/Users/jeromel/Desktop/tmp"
+training_param["output_dir"] = str(pathlib.Path(__file__).parent.absolute())
```

Comment:
Could you wrap all of this code in a `__main__` block and use `argparse` to allow the user to specify the output? I get that this is just an example, but I'm a bit leery of public-facing code that automatically chooses an output directory without user input.